mcs/class/System.Private.CoreLib/System.Collections.Generic/Comparer.cs

```diff
@@ -1,5 +1,7 @@
 using System.Runtime.CompilerServices;
 
+#nullable enable
```

Comment:
nit: should be the first code line in the file. I don't think we have to add it now as the feature will can be enabled globally for the project soon

Reply:
without it I get `error CS0265: Partial declarations of 'ArraySortHelper<T>' have inconsistent constraints for type parameter 'T'` when I try to build S.P.Corlib. (I am building it with .net core 3.0.100-preview6-011673)

Reply:
i mean yeah I understand that a global switch will help but the error text is weird